src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/CreateGithubIssueTest.cs

```diff
@@ -0,0 +1,149 @@
+            secondError.Object.Success = false;
+            secondError.Object.ErrorMessage = "ProcessPendingUpdatesAsync error";
+            secondError.Object.Action = "Create another issue";
+
+            Context.RepoBranchUpdateInMemory = new List<RepositoryBranchUpdateHistoryEntry>
+                {firstError.Object , secondError.Object};
+
+            Mock<Octokit.Repository> repository = new Mock<Repository>();
+            GithubClient.Setup(x => x.Repository.Get(It.IsAny<string>(), It.IsAny<string>())).ReturnsAsync(repository.Object);
+            Mock<Octokit.Issue> issue = new Mock<Issue>();
+            List<NewIssue> newIssue = new List<NewIssue>();
+            GithubClient.Setup(x => x.Issue.Create(It.IsAny<long>(), Capture.In(newIssue))).ReturnsAsync(issue.Object);
+            DependencyUpdateErrorProcessor errorProcessor =
+                ActivatorUtilities.CreateInstance<DependencyUpdateErrorProcessor>(Scope.ServiceProvider,
+                    Context);
+            await errorProcessor.ProcessDependencyUpdateErrorsAsync();
+            string firstIssueTitle =
+                "[Dependency Update] Errors during dependency updates to : https://github.com/maestro-auth-test/maestro-test2";
+            string secondIssueTitle =
+                "[Dependency Update] Errors during dependency updates to : https://github.com/maestro-auth-test/maestro-test3";
+            string firstIssueBody =
+                $@"The following errors have been detected when attempting to update dependencies in 
+'https://github.com/maestro-auth-test/maestro-test2'
+
+ [marker]: <> (subscriptionId: '', method: 'ProcessPendingUpdatesAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+**Repository :** 'https://github.com/maestro-auth-test/maestro-test2'
+**Branch Name :** '38'
+**Error Message :**  'build: 14	Unexpected error processing action: Validation Failed'
+**Method :**   'ProcessPendingUpdatesAsync'
+**Action :**  'Creating new issue'
+**Last seen :**  '1/1/2200 12:00:00 AM'
+**/FyiHandle :** @epananth";
+            string secondIssueBody =
+                $@"The following errors have been detected when attempting to update dependencies in 
+'https://github.com/maestro-auth-test/maestro-test3'
+
+ [marker]: <> (subscriptionId: '', method: 'ProcessPendingUpdatesAsync', errorMessage: 'ProcessPendingUpdatesAsync error')
+**Repository :** 'https://github.com/maestro-auth-test/maestro-test3'
+**Branch Name :** '38'
+**Error Message :**  'ProcessPendingUpdatesAsync error'
+**Method :**   'ProcessPendingUpdatesAsync'
+**Action :**  'Create another issue'
+**Last seen :**  '1/1/2200 12:00:00 AM'
+**/FyiHandle :** @epananth";
+            Assert.Equal(2, newIssue.Count);
+            Assert.Equal("DependencyUpdateError", newIssue[0].Labels[0]);
+            Assert.Equal(firstIssueTitle, newIssue[0].Title);
+            Assert.Equal(firstIssueBody,newIssue[0].Body);
```

Comment:
I wouldn't assert the entire text.  It's highly unlikely the text is going to change on accident.  All this will do it make it a pain to change the opened issues, since you'll have to find all these tests that assert the entire text string and just copy/paste the right answer into them.

If there are specific _parts_ of the text that are programmatically interesting, asserting the body contains them is fine. Like making sure it's got the right date embedded, or the right method, or the right repository url.  But only if they calculation of them is somehow interesting. :-)  I'd rather under-assert big human readable strings than over-assert, because it makes updating them later to be clear much, much harder.